eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php

```diff
@@ -173,4 +210,39 @@ private function getInstaller($type)
 
         return $this->installers[$type];
     }
+
+    /**
+     * Executes a Symfony command in separate process.
+     *
+     * Typically usefull when configuration has changed, our you are outside of Symfony context (Composer commands).
+     *
+     * Based on {@see \Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::executeCommand}.
+     *
+     * @param OutputInterface $output
+     * @param string $cmd eZ Platform command to execute, like 'ezplatform:solr_create_index'
+     *               Escape any user provided arguments, like: 'assets:install '.escapeshellarg($webDir)
+     * @param int $timeout
+     */
+    private function executeCommand(OutputInterface $output, $cmd, $timeout = 300)
```

Comment:
@bdunogier FYI This could also be the answer to allow installers to execute commands after cache has been cleared.